main.py

```diff
@@ -43,14 +45,14 @@ def get_person(person_id, resource):
         return jsonify(person.json_repr(graph))
 
     # Request specific resource associated with the person
-    if resource not in [CIRCLES, EVENTS, KNOWS]:
+    if resource not in [CIRCLES, EVENTS, PEOPLE]:
```

Comment:
Actually better way to do this would just be check 
`if resource == CIRCLES:`
`elif resource == EVENTS:`
`elif resource == PEOPLE:`

and then after all these if statements, just:
`abort(404, description='Invalid resource specified')`

because if it didn't hit any of the if statements and return, then that implies an invalid resource was specified. Could you change this for the other GET requests too please?